src/python/zquantum/core/estimator.py

```diff
@@ -97,6 +97,7 @@ def get_estimated_expectation_values(
                     measurements.get_expectation_values(frame_operator)
                 )
             )
+            print(f"Expectation values for {frame_operator}: {expectation_values_set[-1].values}")
```

Comment:
Do you want to leave it here or is it just some debug code that should disappear?

Reply:
I just removed it. We had been wanting this for some experiments but probably better to leave it out so we don't flood the logs.